src/Features/Core/Portable/ExternalAccess/Pythia/Api/PythiaCompletionProviderBase.cs

```diff
@@ -53,6 +53,9 @@ public static Task<CompletionDescription> GetDescriptionAsync(CompletionItem ite
         public static CompletionDescription GetDescription(CompletionItem item)
             => CommonCompletionItem.GetDescription(item);
 
+        public static bool GetShouldProvideParenthesisCompletion(CompletionItem item)
```

Comment:
Not sure if you added this because you saw my (now deleted) comment, but I think this is actually unnecessary, since intelliccode controls both creation of the item and create the change